Cargo.toml

```diff
@@ -58,9 +57,8 @@ harness = false
 required-features = ["std"]
 
 [features]
-default = ["std", "simd_backend"]
```

Comment:
Is there a reason to remove this from the default features?

Reply:
That's because this feature was deprecated.

version 3.2.0 has feature simd_backend.
https://github.com/dalek-cryptography/curve25519-dalek/blob/3.2.0/Cargo.toml#L71
but version 4.1.1 doesn't have simd_backend.
https://github.com/dalek-cryptography/curve25519-dalek/blob/curve25519-4.1.1/curve25519-dalek/Cargo.toml#L65